### DISCUSSION TITLE
Add zoomable IIIF support

### DISCUSSION THREAD

**Discussion by metasj** [knowledgefutures/pubpub#1376] [Ideas]
Something like the [Universal Viewer](https://universalviewer.io/) supports zoomable viewing a wide range of file types via IIIF.

**What's your request?**
Extend the media embedding options to include something like this for media with an IIIF manifest.

As an editor and reader, this would allow for some beautiful experiences.

**Comment by gabestein** [comment]
Hi everyone — I'm going in and converting some of the longer-lived feature request/idea issues from Issues to Discussion threads. There's no exact heuristic about what belongs in the forum vs. what belongs as an issue, so when in doubt, please post wherever you see fit — we won't mind! But in general, we'd like to make Issues a space for things that we're committed to building/fixing, and Discussions a space for most everything else. So, if you're reporting a bug or requesting we make a straightforward change to an existing feature, go ahead and log an issue. Otherwise, Discussions is likely a better bet.

So, in this case, we're moving it to discussions and inviting anyone who's also interested in the idea to comment!

**Comment by micahvandegrift** [comment]
Interested in this thread. We struggled a bit with embedding various media in Fermentology, and I'd love to offer reader/viewers the ability to engage with images. We have a Storiiies and Internet Archive doc [embedded here](https://fermentology.pubpub.org/pub/csmaf5s3/release/5), and one from HathiTrust [here](https://fermentology.pubpub.org/pub/2g0rujqz/release/6).

**Comment by metasj** [comment]
Revisiting this : IIIF is fascinating; there's a small network of enthusiasts, many of whom are fairly technical and deeply committed to the medium, and the experience of a good zoomable image is magical.  Yet it's quite rare to come across implementations of things like UniversalViewer in the wild.  

We should try it!  

@micahvandegrift, other than zooming, what other engagement would you like?  Are you happy with the Internet Archive interface (other than the nuisance of having to upload there and then embed)?

My ideal would be using one of these [IIIF book-readers](https://labs.cogapp.com/iiif/iiif-book-reader.html) which could work both for single images and for books (paged PDFs w/ searchable text), and has some affordance for annotation.  Mirador has [thought the most about it](https://github.com/ProjectMirador/mirador-annotations).

**Comment by kyledenlinger** [comment]
Adding my +1 here. Currently working with KFG on a project that would benefit from a built-in IIIF viewer. At present, we're going to rely on using the demo/hosted version of [Universal Viewer](http://universalviewer.io/) to generate an embeddable iframe. Right there are two dependencies that will (likely) fail at some point. 

My ideal:
- point IIIF viewer to single documents or entire books
- set custom zoom level & page location for each embed
- toggle tools/viewer features on and off (contents/annotations/etc.)

**Comment by kyledenlinger** [comment]
Coming back here to add an example. Getty's [Quire](https://quire.getty.edu/) digital publishing platform [has IIIF integration for deep zoom](https://quire.getty.edu/docs-v1/zooming-images/) on images. [This chapter](https://www.getty.edu/publications/mummyportraits/part-one/4/) is a good example.

It appears that they're using Leaflet.js for all images, including zoomable IIIF images. An implementation like theirs would also take care of my other request for modal image viewing #2304

**Reply by metasj** [reply]
Thanks for pointing this out -- that's a beautiful example.  What I appreciate about IIIF users, however limited in number, is how exquisite they can be and the attention to detail in communities that prioritize it.
